plugins/kernel_v2/config/vm.rb

```diff
@@ -37,6 +37,9 @@ class VMConfig < Vagrant.plugin("2", :config)
       attr_accessor :usable_port_range
       attr_reader :provisioners
 
+      # This is an experimental feature that isn't public yet.
+      attr_accessor :clone
```

Comment:
We don't document them for anything else. I think its okay. These are usually documented elsewhere.